### PR TITLE
Create Jolokia and Prom JMX exporter directories based on jar path

### PR DESCRIPTION
**Pull request by jeqo**
# Description

Directories to place JARs are being created with hard-coded paths. Instead jar_paths can be used to create a proper directory if variables change.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: roles/confluent.common/tasks/main.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
 - name: Create Jolokia directory
   file:
-    path: /opt/jolokia
+    path: "{{ jolokia_jar_path | name }}"
```
Comment (domenicbove) [inline roles/confluent.common/tasks/main.yml:48]:
```suggestion
    path: "{{ jolokia_jar_path | dirname }}"
```

**Review by domenicbove — APPROVED** [review]
Great catch! one minor typo

**Review by JumaX — APPROVED** [review]
LGTM.

**Review by JumaX — CHANGES_REQUESTED** [review]
@jeqo Can you rebase this to 5.5.x, as we should fix it there as well.

**Review by JumaX — APPROVED** [review]
LGTM!

**Pull request merged**